deploy/9010_set_sand_admin.ts

```diff
@@ -17,17 +17,21 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     );
   }
 
-  const currentExecutionAdmin = await read('Sand', 'getExecutionAdmin');
-  if (
-    currentExecutionAdmin.toLowerCase() !== sandExecutionAdmin.toLowerCase()
-  ) {
-    log('setting Sand Execution Admin');
-    await execute(
-      'Sand',
-      {from: currentExecutionAdmin, log: true},
-      'changeExecutionAdmin',
-      sandExecutionAdmin
-    );
+  const sand = await ethers.getContract('Sand');
+  if (sand.callStatic.changeExecutionAdmin) {
```

Comment:
How come you changed this to `callStatic`?

Reply:
Basically the SAND contract deployed on rinkeby had the function `changeExecutionAdmin` called `changeExecutionAdminAdmin` so here I just check if that function exists, as otherwise it would fails on the rinkeby SAND